sos/report/plugins/multipath.py

```diff
@@ -25,7 +25,8 @@ def setup(self):
             "multipath -ll",
             "multipath -v4 -ll",
             "multipath -t",
-            "multipathd show config"
+            "multipathd show config",
+            "lvm pvdisplay"
```

Comment:
This should go in the `lvm2` plugin, not `multipath`.

Reply:
I think existing commands inside `lvm2` already  `pvdisplay` output details.  Just that all output is not in one place.

Reply:
Thanks Jake and Vasant for review comments,

Vasant,
As you have mentioned lvm2 plugin already has pvdisplay ouput  details,
looks like we don't need this data